content/Kotlin/kotlin.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'Kotlin'
+Description: 'Kotlin is a contemporary language released by JetBrains in 2011. It is becoming popular and quickly replacing Java as the choice for Android software projects.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+[Kotlin](https://www.codecademy.com/catalog/language/kotlin) is a contemporary language released by JetBrains in 2011. Kotlin is becoming popular and is fast replacing Java as people’s choice for software projects. Java and Kotlin are compatible and proficiency in Java makes learning Kotlin much easier. 
```

Comment:
```suggestion
[**Kotlin**](https://www.codecademy.com/catalog/language/kotlin) is a programming language used for general purposes including [front-end](https://www.codecademy.com/resources/docs/general/front-end), [back-end](https://www.codecademy.com/resources/docs/general/back-end), and mobile development. Since Kotlin code is fully compatible with Java, its syntax may seem familiar to those with proficiency in Java.

After being released by JetBrains in 2011, Kotlin became the preferred language for Android development by Google in 2019.
```

I'm not sure Docs is the right place to state whether one language is replacing another. However, I do think it is fair to say that having a background in Java makes working with Kotlin relatively easier. 😄